### PR TITLE
Adding custom scalars

### PR DESCRIPTION
**Pull request by marshalWilliam**
Added custom scalars (Binary, EmailAddress, DateTime)

### CODE REVIEW

File: schemas/scalars/datetime.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  parseValue(value: any) {
+    return new Date(value);
+  },
+  parseLiteral(ast) {
+    if (ast.kind !== Kind.STRING) {
+      throw new GraphQLError(
+        `Query Error: Can only parse strings as datetime but got a: ${ast.kind}`
+      );
+    }
+
```
Comment (ccpacillos) [inline schemas/scalars/datetime.ts:23]: Add validation here to check if input string is a valid ISO date string.

**Review by ccpacillos — CHANGES_REQUESTED** [review]

File: schemas/scalars/datetime.ts (hunk 1 above)
Comment (marshalWilliam) [reply schemas/scalars/datetime.ts:23]: Good afternoon po sir, I've added the validation sir. Thank you sir!

File: schemas/scalars/datetime.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    if (ast.value !== new Date(Date.parse(ast.value)).toISOString()) {
```
Comment (ccpacillos) [inline schemas/scalars/datetime.ts]: There's a better way of checking this. You can refer to similar solutions here: https://thewebdev.info/2021/03/27/how-to-check-if-a-string-is-a-date-string-with-javascript

**Review by ccpacillos — CHANGES_REQUESTED** [review]

File: schemas/scalars/datetime.ts (hunk 2 above)
Comment (marshalWilliam) [reply schemas/scalars/datetime.ts]: Noted po sir. How about this one po.

**Review by ccpacillos — APPROVED** [review]

**Pull request merged**